qemu/tests/cfg/pci_devices.cfg

```diff
@@ -2,6 +2,7 @@
     type = pci_devices
     start_vm = no
     pci_controllers = ''
+    pci_controllers_autosort = "no"
```

Comment:
It is actually introduced because of this test. Originally no sorting was being done and devices were added in the order they were defined in `pci_controllers` but (I don't know why) sorting was introduced which probably was suppose to simplify peoples life, but it breaks this test as it makes certain scenarios impossible to define in Avocado-vt. The `pci_controllers_autosort` allows you to use the original behavior.